code/modules/mob/living/simple_animal/abnormality/he/scaredy_cat.dm

```diff
@@ -0,0 +1,165 @@
+	datum_reference.qliphoth_change(1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/scaredy_cat/breach_effect(mob/living/carbon/human/user)
+	if(priority_friend) //if an oz abno escape they take absolute priority
+		ProtectFriend(priority_friend)
+		return ..()
+	var/list/breached_abno = list()
+	for(var/datum/abnormality/A in SSlobotomy_corp.all_abnormality_datums)
+		if(!(A.current.status_flags & GODMODE) && A.name != "Scaredy cat")
+			breached_abno += A.current
+	if(LAZYLEN(breached_abno))
+		ProtectFriend(pick(breached_abno))
+	else
+		wait_for_friend = TRUE //Should only happen on meltdowns, can safely be killed in that state too
+	..()
+
+///checks if the friend is in view every 10 second, and if not teleports to it
+/mob/living/simple_animal/hostile/abnormality/scaredy_cat/Life()
+	. = ..()
+	if(!friend || status_flags & GODMODE || stat == DEAD) //for some reason life() works on death ain't that something
+		return
+	if(protect_cooldown < world.time)
+		protect_cooldown = world.time + protect_cooldown_time
+		if(!can_see(src, friend, vision_range))
+			GoToFriend()
+
+/mob/living/simple_animal/hostile/abnormality/scaredy_cat/proc/ProtectFriend(mob/living/simple_animal/hostile/abnormality/abno)
+	wait_for_friend = FALSE
+	Courage(TRUE)
+	friend = abno //You are friend =D
+	faction = friend.faction
+	GoToFriend()
+
+/mob/living/simple_animal/hostile/abnormality/scaredy_cat/death()
+	density = FALSE
+	if(friend)
+		addtimer(CALLBACK(src, .proc/Regenerate), 20 SECONDS)
+	else
+		animate(src, alpha = 0, time = 10 SECONDS)
+		QDEL_IN(src, 10 SECONDS)
+	..()
+
+/mob/living/simple_animal/hostile/abnormality/scaredy_cat/Destroy()
+	UnregisterSignal(SSdcs, COMSIG_GLOB_MOB_DEATH)
+	UnregisterSignal(SSdcs, COMSIG_GLOB_ABNORMALITY_BREACH)
+	return ..()
+
+//if he still has a friend, revives
+/mob/living/simple_animal/hostile/abnormality/scaredy_cat/proc/Regenerate()
+	if(friend)
+		density = TRUE
+		revive(full_heal = TRUE, admin_revive = TRUE)
+		GoToFriend()
+		return
+	animate(src, alpha = 0, time = 3 SECONDS)
+	QDEL_IN(src, 3 SECONDS)
+
+///teleports to his friend if he can't see them
+/mob/living/simple_animal/hostile/abnormality/scaredy_cat/proc/GoToFriend()
+	if(!friend)
+		return
+	var/turf/origin = get_turf(friend)
+	var/list/all_turfs = RANGE_TURFS(2, origin)
+	for(var/turf/T in all_turfs)
+		if(!T.density && T != origin && can_see(origin, T, friend.vision_range))
+			playsound(src, 'sound/abnormalities/scaredycat/cateleport.ogg', 50, FALSE, 4)
+			forceMove(T)
+			for(var/mob/living/carbon/human/enemy in view(vision_range, src))
+				GiveTarget(enemy) //the moment he teleports he's already on the offensive
+				return
+
+///If scaredy cat becomes a big boy or a baby boy
+/mob/living/simple_animal/hostile/abnormality/scaredy_cat/proc/Courage(courage)
+	if(courage)
+		melee_damage_lower = 20
+		melee_damage_upper = 25
+		damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.5, WHITE_DAMAGE = 2, BLACK_DAMAGE = 1.5, PALE_DAMAGE = 2)
```

Comment:
I know it can come back but it has a MEASILY 400 health overall. That should be increased OR these should be increased. Or even a combination of both. Either-way, he's kinda very squishy here